datahub-gms-graphql-api/src/main/java/com/linkedin/datahub/graphql/api/GmsGraphQLProvider.java

```diff
@@ -0,0 +1,34 @@
+package com.linkedin.datahub.graphql.api;
+
+import javax.annotation.PostConstruct;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.stereotype.Component;
+
+import com.linkedin.datahub.graphql.GmsGraphQLEngine;
+import com.linkedin.datahub.graphql.GraphQLEngine;
+
+import graphql.GraphQL;
+
+@Component
+public class GmsGraphQLProvider {
+
+    private GraphQL graphQL;
+    private GraphQLEngine graphQLEngine;
+
+    @Bean
+    public GraphQL graphQL() {
+        return graphQL;
+    }
+
+    @Bean
+    public GraphQLEngine graphQLEngine() {
```

Comment:
Do we need to explicitly provide the `GraphQLEngine` in addition to the `GraphQL` instance, or is the`GraphQL` instance enough?

Reply:
As`GraphQLEngine` is used by the class `QueryGraphQLInvocation` it needs to be provided to be Autowired in the Invocation class. 
Let me know if there is a better way to do this.